src/sentry/api/serializers/models/exploresavedquery.py

```diff
@@ -65,7 +66,13 @@ def get_attrs(self, item_list, user, **kwargs):
             result[explore_saved_query]["created_by"] = serialized_users.get(
                 str(explore_saved_query.created_by_id)
             )
-            result[explore_saved_query]["starred"] = explore_saved_query.id in starred_query_ids
+            result[explore_saved_query]["starred"] = explore_saved_query.id in starred_queries
```

Comment:
If you want to use the if/else statements below, you can drop this assignment right?

Reply:
left this in by mistake, thanks!